app/src/main/java/com/example/cinemapp/data/MovieRemoteDataSource.kt

```diff
@@ -130,6 +135,35 @@ interface MovieRemoteDataSource {
         @Query("sort_by") sortBy: String = "created_at.desc"
     ): Response<MovieResponseDTO>
 
+    @POST("$API_VERSION/account/account_id/favorite")
+    suspend fun setFavorite(
+        @Query("session_id") sessionId: String,
+        @Body body: SetFavoriteBodyDTO
+    ): Response<StatusResponseDTO>
+
+    @POST("$API_VERSION/account/account_id/watchlist")
```

Comment:
Does this API call work? Should a part of this path be `account/{account_id}` (like in the API call below), or this is how it's documented on TMDB API?

Reply:
When making this network call, you can send either the account id or the session id. Since i am using the session id, that field is supposed to be left like that.